### DISCUSSION TITLE
Got 404 error on github page

### DISCUSSION THREAD

**Discussion by Hisa12** [community/community#59996] [Other Feature Feedback, Questions, & Ideas] [answered]
Question

Hi there. I've faced 404 error when I deployed my website to github. 
I've followed many instructions but they haven't solved the error yet.
Could anyone help me with that?

this is my repo.
https://github.com/Hisa12/portfolio.git

**Comment by 3urobeat** [comment]
Your page is reachable under https://hisa12.github.io/portfolio/index.html as expected.  
The 404 error displayed on your page might be caused by the `src` paths in your [index.html](https://raw.githubusercontent.com/Hisa12/portfolio/gh-pages/index.html) as it cannot find the referenced files.  
I think you need to remove every `/portfolio/` from every `src` parameter as the `static` folder is already in the same folder as `index.html`.

**Reply by Hisa12** [reply]
Thanks for your response. Unfortunately, https://hisa12.github.io/portfolio/index.html is not reachable and it remain 404 even though I deleted /portfolio/ from 'src= '/portfolio/...'

**Reply by 3urobeat** [reply]
The [index.html](https://raw.githubusercontent.com/Hisa12/portfolio/gh-pages/index.html) in your gh-pages branch still contains `/portfolio/` in a lot of paths

**Reply by Hisa12** [reply]
I got these errors when I deleted '/portfolio/' in index.html.
<img width="673" alt="Screen Shot 2023-07-04 at 21 34 13" src="https://github.com/community/community/assets/94655243/c3d79d94-e314-4176-b961-c7ed6e27005a">

**Comment by GhostOf0days** [comment, marked as answer]
I see someone else is helping you here, so I'll chime in as future advice rather than a solution here. You should use something like Netlify or Vercel instead of Github Pages. For simple use cases/websites, use Netlify.

**Reply by Hisa12** [reply]
I could successfully display my portfolio on Netlify. Thanks for your advice!

**Reply by SYAH0407** [reply]
/

**Comment by Martin-IJ** [comment]
I'm getting this error message each time i try signing in to my Netlify account -
Authentication Error
Authenticating failed due to the following error: We already have a registered user with this email address. Log in to connect your GitHub account.

How do I resolve this please
<img width="655" alt="Screenshot 2023-08-03 at 1 25 33 PM" src="https://github.com/community/community/assets/66225932/45a7bd8a-803b-49ec-8c69-3e49458d5fed">
